app/menus/menus/help.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (os, app, shell) {
+  return {
+    role: 'help',
+    submenu: [
+      {
+        label: `${app.getName()} Website`,
+        click() {
+          shell.openExternal('https://hyper.is');
+        }
+      },
+      {
+        label: 'Report Issue',
+        click() {
+          const body = `
+          <!-- Please succinctly describe your issue and steps to reproduce it. -->
```

Comment:
Would be awesome if this templated in the issue template: https://raw.githubusercontent.com/zeit/hyper/master/.github/issue_template.md

not needed for this PR though 😄  just an idea